README.md

```diff
@@ -26,56 +28,71 @@ npm start
 
 Browse http://localhost:8080 and open the dev tools console to see the proof being generated and its verification
 
-
 ### Usage
 
 #### Initializing the library
+
 ```js
 import * as rln from "@waku/rln";
 
 const rlnInstance = wait rln.create();
 ```
 
 #### Generating RLN membership keypair
+
 ```js
 let memKeys = rlnInstance.generateMembershipKey();
 ```
 
+#### Generating RLN membership keypair using a seed
+
+```js
+let memKeys = rlnInstance.generateSeededMembershipKey(seed);
+```
 
 #### Adding membership keys into merkle tree
+
 ```js
 rlnInstance.insertMember(memKeys.IDCommitment);
 ```
 
 #### Generating a proof
+
 ```js
 // prepare the message
-const uint8Msg = Uint8Array.from("Hello World".split("").map(x => x.charCodeAt()));
+const uint8Msg = Uint8Array.from(
+  "Hello World".split("").map((x) => x.charCodeAt())
+);
```

Comment:
This only works with ASCII AFAIK, I wonder if we could just use https://developer.mozilla.org/en-US/docs/Web/API/TextEncoder (I think I tried in the paste but don't remember exactly).